### PR TITLE
Task/bws131 add id and ip address to instance table

### PR DESCRIPTION
**Pull request by tbezrukava**

### CODE REVIEW

**Comment by tbezrukava** [comment]
Снимок экрана 2022-03-14 в 18 04 34

File: frontend/package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "@uiw/react-codemirror": "4.5.1",
     "bws-shared": "file:../shared/build",
     "clsx": "1.1.1",
+    "copy-to-clipboard": "3.3.1",
```
Comment (what1s1ove) [inline frontend/package.json]:
```suggestion
```
https://developer.mozilla.org/en-US/docs/Web/API/Navigator/clipboard

File: frontend/src/assets/img/key-icon.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg fill="none" xmlns="http://www.w3.org/2000/svg"  viewBox="0 0 50 50" width="20px" height="20px">
```
Comment (what1s1ove) [inline frontend/src/assets/img/key-icon.svg]:
```suggestion
<svg fill="none" xmlns="http://www.w3.org/2000/svg"  viewBox="0 0 50 50" width="20" height="20">
```

File: frontend/src/components/sc/sc.tsx [hunk 3]
```diff
@@ -18,18 +22,37 @@ const SC: FC = () => {
...
+    if (sshKey) {
+      copy(sshKey.sshKey, {
+        message: 'copied',
+      });
+    }
+
+    return (): void => {
+      dispatch(scActions.cleanupSshKey());
+    };
+  }, [sshKey, dispatch]);
```
Comment (what1s1ove) [inline frontend/src/components/sc/sc.tsx]:
```suggestion
```
do copy logic in the action after getting the key from server

File: frontend/src/store/sc/reducer.ts [hunk 4]
```diff
@@ -1,16 +1,26 @@
...
+  deleteInstance,
+  loadInstances,
+  loadSshKey,
+  cleanupSshKey,
+} from './actions';
 
 type State = {
   dataStatus: DataStatus;
   instances: SCInstanceGetByTenantResponseItemDto[];
+  sshKey: SCSshKeyGetByIdResponseDto | null;
```
Comment (what1s1ove) [inline frontend/src/store/sc/reducer.ts]:
```suggestion
```
just copy in the clipboard in the action. you don't need it in the store

**Review by what1s1ove — COMMENTED** [review]
remove `frontend/src/assets/img/icons8-key-2-20.png`

File: frontend/src/store/sc/reducer.ts (hunk 4 above)
Comment (what1s1ove) [reply frontend/src/store/sc/reducer.ts]: isn't fixed

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Pull request merged**